lib/rucio/web/rest/flaskapi/v1/archive.py

```diff
@@ -54,7 +58,7 @@ def get(self, scope, name):
                 data += dumps(file) + '\n'
             return Response(data, content_type="application/x-json-stream")
         except Exception, error:
```

Comment:
shouldn't this be `except Exception as error` for py3? (there's three occurrences like this in this merge request)